apps/studio/src/pages/sites/[siteId]/collections/[resourceId].tsx

```diff
@@ -34,7 +42,27 @@ const CollectionResourceListPage: NextPageWithLayout = () => {
   return (
     <>
       <Stack w="100%" p="1.75rem" gap="1rem">
-        {/* TODO: Add breadcrumb */}
+        <Breadcrumb size="sm">
+          <BreadcrumbItem>
+            <BreadcrumbLink href={`/sites/${siteId}`}>
+              <Text textStyle="caption-2" color="interaction.links.default">
+                Home
+              </Text>
+            </BreadcrumbLink>
+          </BreadcrumbItem>
+          <BreadcrumbItem>
+            <Text textStyle="caption-2" color="base.content.default">
+              ...
```

Comment:
These are fake breadcrumbs yeah? Any issue tracked to have the actual crumbs? Might be hard with our child -> parent link

Reply:
adding issue! clarified w/ sehyun, technically supposed to show all items between home and current page on hover monkas

Reply:
added issue